app/views/vacancies/_show.html+phone.haml

```diff
@@ -32,12 +32,15 @@
 
     %table.check-your-answers.cya-questions-short
       %tbody
-        %tr
-          %td.cya-question= t('jobs.salary')
-          %td.cya-answer= @vacancy.salary_range
         %tr
           %td.cya-question= t('jobs.working_pattern')
           %td.cya-answer= @vacancy.working_pattern
+        %tr
+          %td.cya-question= t('jobs.salary')
+          %td.cya-answer
+            = @vacancy.salary_range
+            - if @vacancy.part_time?
+              pro rata
```

Comment:
All the other copy is managed by locale, not a blocker but perhaps this was missed?